mmv1/third_party/terraform/tests/resource_network_services_mesh_test.go.erb

```diff
@@ -0,0 +1,63 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
```

Comment:
This should be able to be a `.go` file instead of `.go.erb` file. 

`.go.erb` is only needed when you have version specific requirements or need other ruby related features. 

Notice the line `<% unless version == 'ga' -%>`.
This is stating to only emit this test in the ga provider. To my knowledge it should be okay to have this test in the `ga` and `beta` providers. Thus we should be able to change the file type from `.go.erb` to `.go` and remove any of the ruby related code (enclosed by `<%` and `%>`)

Reply:
This is needed to generate the tests only for beta. It is saying skip for ga and not the other way around (I found this confusing too). We are following the instructions here:
https://googlecloudplatform.github.io/magic-modules/docs/how-to/add-handwritten-test/#testing-beta-features

Reply:
The instructions also ask to set the file extension to .go.erb

Reply:
got it !